.github/workflows/funding.yml

```
@@ -0,0 +1 @@
+buy_me_a_coffee: jyourstone
```

actionlint (1.7.9)

"jobs" section is missing in workflow

(syntax-check)